src/PrestaShopBundle/Resources/config/services/bundle/form/form_extension.yml

```
@@ -16,6 +16,20 @@ services:
     tags:
       - { name: form.type_extension, extended_type: Symfony\Component\Form\Extension\Core\Type\MoneyType }
 
+  form.type.extension.number:
+    class: 'PrestaShopBundle\Form\Admin\Type\CustomNumberType'
+    arguments:
+      - '@prestashop.core.localization.locale.context_locale'
+    tags:
+      - { name: form.type_extension, extended_type: Symfony\Component\Form\Extension\Core\Type\NumberType }
+
+  form.type.extension.integer:
+    class: 'PrestaShopBundle\Form\Admin\Type\CustomIntegerType'
+    arguments:
+      - '@prestashop.core.localization.locale.context_locale'
+    tags:
+      - { name: form.type_extension, extended_type: Symfony\Component\Form\Extension\Core\Type\IntegerType }
```

According to ADR 19, you should use FQCN https://github.com/PrestaShop/ADR/blob/master/0019-FQCN-Autowiring.md, no ?

```suggestion
  PrestaShopBundle\Form\Admin\Type\CustomNumberType:
    public: false
    arguments:
      - '@prestashop.core.localization.locale.context_locale'
    tags:
      - { name: form.type_extension, extended_type: Symfony\Component\Form\Extension\Core\Type\NumberType }

  PrestaShopBundle\Form\Admin\Type\CustomIntegerType:
    public: false
    arguments:
      - '@prestashop.core.localization.locale.context_locale'
    tags:
      - { name: form.type_extension, extended_type: Symfony\Component\Form\Extension\Core\Type\IntegerType }
```